internal/services/blackwhitelist_test.go

```diff
@@ -86,151 +84,205 @@ func TestGenerateProjects(t *testing.T) {
 			Project:     "native",
 			Environment: "development",
 		},
+		{
+			Project:     "native",
+			Environment: "integration",
+		},
+		{
+			Project:     "native",
+			Environment: "production",
+		},
 	}
 
+
+
```

Comment:
nit: missing a whitelist_enabled and no projects in blacklist/whitelist for example. can you do it in a follow up commit?